README.md

```diff
@@ -32,8 +32,48 @@ dependencies {
 ```
 <!-- x-release-please-end -->
 
+
+## Releases
+
+The Path SDK is published via [JitPack](https://jitpack.io/#mxenabled/path-sdk).
+
+### Branching Strategy
+
+#### master
+
+`master` contains the edge development for the current major version. All code merged into `master` should be complete, tested, and releasable (from a feature branch).
+
+#### feature branches
+
+  1. Developers create feature branches to introduce code changes
+  2. When complete, a Pull Request is created from the branch
+  3. Once approved, the pull request is merged into `master`
+
+#### release branches
+
+When work on a _planned_ major version is ready to start, a release branch will be created to hold the in-progress the work. Multiple feature branches and pull requests will be generated against this branch until it the version is ready for release.
```

Comment:
```suggestion
When work on a _planned_ major version is ready to start, a release branch will be created to hold the in-progress work. Multiple feature branches and pull requests will be generated against this branch until the version is ready for release.
```
Grammar

Reply:
I don't know how many times I've read this. Still missed those: 🤦

Reply:
Fixed